plugin/trino-druid/src/test/java/io/trino/plugin/druid/DruidQueryRunner.java

```diff
@@ -112,7 +113,7 @@ private static void writeDataAsTsv(MaterializedResult rows, String dataFile)
             throws IOException
     {
         File file = new File(dataFile);
-        try (BufferedWriter bw = new BufferedWriter(new FileWriter(file))) {
+        try (BufferedWriter bw = new BufferedWriter(new FileWriter(file, UTF_8))) {
```

Comment:
this is processing a test resource, right?

Reply:
It's more like a local file, doesn't look like it's a committed resource:

```java
String tsvFileLocation = format("%s/%s.tsv", testingDruidServer.getHostWorkingDirectory(), druidDatasource);
writeDataAsTsv(rows, tsvFileLocation);
```

Reply:
sorry, it's a write, so it doesn't exist yet

i don't know what it should be (how Druid reads this) but let's use utf-8 here. this is tests anyway, and tests have fixed utf-8 charset afaict

Reply:
> it's a write, so it doesn't exist yet

Oh. You're right, of course :)